src/ProxyManager/GeneratorStrategy/FileWriterGeneratorStrategy.php

```diff
@@ -44,16 +37,12 @@ public function generate(ClassGenerator $classGenerator): string
         $className     = $classGenerator->getNamespaceName() . '\\' . $classGenerator->getName();
         $fileName      = $this->fileLocator->getProxyFileName($className);
 
-        set_error_handler($this->emptyErrorHandler);
```

Comment:
Discussed in chat: change is valid, I just want to merge the removal of the error handlers separately from the test adjustment first, but overall 👍